src/pages/projects/ProjectConfigurationHeader.tsx

```diff
@@ -47,28 +48,44 @@ const ProjectConfigurationHeader: FC<Props> = ({ project }) => {
         formik.setSubmitting(false);
         return;
       }
+      const projectLink = (
+        <ResourceLink
+          type="project"
+          value={values.name}
+          to={`/ui/project/${project.name}/configuration`}
```

Comment:
This is confusing, because it is linking to the old url. Maybe give the variable holding it a better name?

Reply:
yeah you are right, calling it `oldProjectLink`.